### DISCUSSION TITLE
I want `srcset` and `sizes` to relate to the container rather than the viewport

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#61] [Wants]
Currently, images with `srcset` and `sizes` attributes can only be used relative to the viewport. When you have a more dynamic design where images are different widths depending on the content, you’d like `srcset` to relate to the container they’re in, and not the viewport.

<hr>

https://webwewant.fyi/wants/5f1aa74a3bfb5b5fff803e91/

**Comment by nhoizey** [comment]
It would be nice, but I don't think browsers would be able to decide which image to load before the rendering is done, which would be much later than current feature.

**Reply by aarongustafson** [reply]
Yeah, I would love to hear from @bkardell and @gregwhitworth on this.
